src/CoreFoundation/CFString.cs

```diff
@@ -124,6 +124,11 @@ public class CFString
 
 		[DllImport (Constants.CoreFoundationLibrary, CharSet=CharSet.Unicode)]
 		extern static IntPtr CFStringGetCharacters (IntPtr handle, CFRange range, IntPtr buffer);
+
+		internal static IntPtr LowLevelCreate (string str)
+		{
+			return CFStringCreateWithCharacters (IntPtr.Zero, str, str.Length);
```

Comment:
Looks easier (maintenance) and smaller to handle `null` in the call (not in the caller) and return `IntPtr.Zero`